config/runtime.exs

```diff
@@ -13,3 +13,13 @@ config :neurow, sse_timeout: String.to_integer(System.get_env("SSE_TIMEOUT") ||
 
 config :neurow, ssl_keyfile: System.get_env("SSL_KEYFILE")
 config :neurow, ssl_certfile: System.get_env("SSL_CERTFILE")
+
+config :neurow,
```

Comment:
How do we inject this in production? Can we define a json format?

Reply:
I will update the code so we have a `dev.exs` that directly contains test keys, and `prod.exs` that parse an environment variable.